students/M_Sunday/lesson08/circle.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+from math import pi
+
+
+class Circle(object):
+
+    @staticmethod
+    def entry_check(the__radius):
+        if isinstance(the__radius, (str, list, tuple, dict)):
```

Comment:
I'd go with a float conversion or some other method to directly raise ValueError on bad input rather than trying to list out all the possible bad types